appdatabase/migrations/sql/1642771235_add_communities_settings.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE communities_settings (
+  community_id TEXT PRIMARY KEY ON CONFLICT REPLACE,
```

Comment:
In protocol/migrations/sqlite/1605075346_add_communities.up.sql  the community_id is defined as 
`  id BLOB NOT NULL PRIMARY KEY ON CONFLICT REPLACE`.  Consider using the same data type (blob)

Reply:
Good call!